EIPS/eip-7336.md

```diff
@@ -0,0 +1,104 @@
+---
+eip: 7336
+title: Interactive across multiple registries
+description: An Universal Domain Namespace(UDoN) interface allowing for interactive capabilities across multiple registries like ENS
+author: Mengshi Zhang (@MengshiZhang), Zihao Chen (@zihaoccc)
+discussions-to: https://ethereum-magicians.org/t/erc7336-a-interface-allowing-for-interactive-capabilities-across-multiple-registries-like-ens/15069
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-07-14
+requires: 165
+---
+
+## Abstract
+
+The objective of this EIP is to propose the **Universal Domain Namespace(UDoN)**, which aims to facilitate onchain registry entries. Each entry in this structure adopts a mapping type, allowing for interactive capabilities across multiple registries like [ENS](./eip-137.md) and [NFT accounts](./eip-6551.md).
```

Comment:
Linking to ERC-6551 creates an implicit relationship between this proposal and that one. That mean ERC-7336 won't be able to advance beyond the status of ERC-6551 (currently Review.) This isn't a problem if it's intended on your part, I just wanted to mention it.